### PR TITLE
Provide 'freeze' function to BEMHTML 'root' block #53

### PR DESCRIPTION
**Pull request by vithar**

### CODE REVIEW

**Comment by vithar** [comment]
//cc @tadatuta Take a look.

File: src/tasks/template/full-bem.js [hunk 1]
```diff
@@ -102,14 +115,8 @@ module.exports = function(model, options) {
...
     function compoundPage(page, content) {
-        if(_.includes(page.contentFile, '.js')) {
-            content = BEMHTML.apply(vm.runInNewContext(content, {
-                freeze: file => {
-                    var absFilePath = path.resolve(path.dirname(page.source), file);
-
-                    return freeze.freeze(absFilePath).replace(path.resolve(options.destinationRoot), '');
-                }
-            }));
+        if(_.includes(page.contentFile, '.bemjson.js')) {
```
Comment (tadatuta) [inline src/tasks/template/full-bem.js]: `_.endsWith()`?

File: src/tasks/template/full-bem.js [hunk 2]
```diff
@@ -126,7 +133,9 @@ module.exports = function(model, options) {
                 .then(getPageContent)
                 .then(compoundPage.bind(null, page))
                 .then(createBEMJSON.bind(null, pages))
-                .then(BEMHTML.apply)
+                .then(function(bemjson){
```
Comment (tadatuta) [inline src/tasks/template/full-bem.js]:
```js
.then(_bemhtml)
```

**Comment by vithar** [comment]
Updated

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**